src/types.rs

```diff
@@ -602,3 +609,37 @@ pub struct FullCrate {
 pub(crate) struct UserResponse {
     pub user: User,
 }
+
+pub(crate) mod helper {
+    use super::{Authors, Dependency, FullVersion, Version};
+
+    /// Helper for conveniently an consistently converting a [`Version`] and the addional data into
+    /// a [`FullVersion`].
+    impl From<(Version, Authors, Vec<Dependency>)> for FullVersion {
```

Comment:
A `From` impl for a tuple like this is quite cryptic for future readers.

Can we turn this into a function like `FullVersion::from_parts` instaed?

Reply:
Good catch. Thank you! The latter is way more readable. 😅